extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -400,7 +436,17 @@ class MembershipsButtonEdit extends Component {
 											'jetpack'
 										) }
 									</p>
-									<Button isDefault isLarge href={ connectURL } target="_blank">
+									<Button
+										isDefault
+										isLarge
+										disabled={ ! stripeConnectUrl }
```

Comment:
`getConnectUrl()` will return `null` if we don't have `postId`, but from `target` attribute I understood like this button should still work even if we're missing `postId`?

Reply:
Thanks, we could allow connecting if we haven't saved the post. I intended to allow that but it's not finished: https://github.com/Automattic/jetpack/pull/13398/files#r320721328

That's probably not ideal and we should return to the saved post or a new if we started in the editor.